### PR TITLE
HW5- Copied Tutorial 3 in personal directory florianjoe

### PR DESCRIPTION
**Pull request by florianjoerg**

### CODE REVIEW

File: florianjoe/homework3/code/musicstats.sh [hunk 1]
```diff
@@ -0,0 +1,310 @@
+#!/bin/bash
```
Comment (floridop) [inline florianjoe/homework3/code/musicstats.sh]: Please delete this file!

File: florianjoe/homework3/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
+--------------------------------------------------------
```
Comment (floridop) [inline florianjoe/homework3/README.md]: Please delete this entire folder homework3!

**Comment by floridop** [comment]
Hi Florian

thanks for you submission. However In this submission I'd like **not** to have the `musictstats.sh` file.
More importantly it seems you double copied the `homework3` folder that adds a lot of noise. I'd prefer you only keep the `tutorial3/homework3` folder and copy your code there, otherwise it's a lot of file repetitions all over.

**Comment by florianjoerg** [comment]
Thank you for the feedback!

Sorry, I was a bit confused by the different braches and where which file should be, I hope I sorted it now. (hw3hw5 branch only having the files from your tutorial3 directory and the HW-tutorial3 branch having only my solution to the musicstats file).

**Pull request merged**